presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ExchangeNode.java

```diff
@@ -57,7 +55,18 @@
     public enum Scope
     {
         LOCAL,
-        REMOTE
+        REMOTE,
```

Comment:
I wonder if this should also have a modifier now, e.g. `REMOTE_STREAMING`

Reply:
`REMOTE_NETWORK` ? :)

Reply:
I think `REMOTE_STREAMING` is a better candidate, because both of them are in some sense over the network.